test/presenters/user_presenter_test.rb

```diff
@@ -23,10 +23,13 @@ def item
   end
 
   test "gravatar id" do
-    assert_equal '72dedd9e525e529e37b724e8aba4997f', pres.gravatar_id
+    pres.save
+    expected = '72dedd9e525e529e37b724e8aba4997f'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.